packages/web/src/header/Header.3.tsx

```diff
@@ -164,13 +150,40 @@ export class Header extends React.PureComponent<Props, State> {
     this.setState({ bannerHeight: height })
   }
 
+  allWhiteLogo = () => {
+    return this.props.router.pathname === menu.ABOUT_US.link && !this.state.belowFoldUpScroll
```

Comment:
This seems hacky, is there a better way that doesn't hardcode in a pathname?

Reply:
I could define a Set of all path names which should use an all white logo? at module scope